src/apps/frontend/types/auth.ts

```diff
@@ -3,10 +3,22 @@ import { JsonObject } from './common-types';
 export class AccessToken {
   accountId: string;
   token: string;
+  expiresAt:Date;
 
   constructor(json: JsonObject) {
     this.accountId = json.account_id as string;
     this.token = json.token as string;
+    // Convert from Unix timestamp (seconds) to Date object
+    this.expiresAt = new Date(parseFloat(json.expires_at as string) * 1000);
```

Comment:
Too many conversions, `as string`, `parseFloat`, and then multiply by 1000.

Ideally, server should return the value is ISO 8061 format, and Date.parse should handle it.